views/index.erb

```diff
@@ -0,0 +1,20 @@
+<!-- views/index.erb -->
+<html>
+  <head>
+    <title>Animals that freak me out in a good way</title>
+    <meta charset = "utf-8">
+  </head>
+
+  <body>
+    <img src= "Linny-pig.jpg" width="50%" height="69%"/>
+    <h1>Animals that freak me out in a good way</h1>
+
+    <nav>
+      <ul>
+        <li><a href="http://localhost:9292/dugong">Dugong</a></li>
```

Comment:
You don't want to use your whole `localhost` URL in the link especially b/c if/when you deploy it, it will be wrong. A relative path would be better. Then you would need to add routes in your my-site.rb file to handle these pages as well.